test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/scaling/KieServerWebSocketScalingIntegrationTest.java

```diff
@@ -50,7 +51,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@Category({Baseline.class, OperatorNotSupported.class}) // Operator doesn't contain any scenario with controller based strategy by default
+@Category({Baseline.class, OperatorNotSupported.class, MonitoringK8sFs.class}) // Operator doesn't contain any scenario with controller based strategy by default
```

Comment:
Is this correct? This integration test will be executed in Templates only and the MonitoringK8sFs is only for Operator.

Reply:
It may looks bad (and I must admit that I didn't realize that), but I would leave this configuration here. Right now this will be skipped as it is not supported in Operator, but in future we may allow some of the tests for Operator and we could forgot to set MonitoringK8sFs tests.
If we see that this configuration is in conflict in tests, we can remove it later.

Reply:
Ok ok